index.html

```diff
@@ -0,0 +1,55 @@
+<!DOCTYPE html>
+<html>
+<head>
+	<title>MrBlue</title>
+	<link rel="stylesheet" href="static/style.css">
+	<meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    
+</head>
+<body>
+	<header class="header">
+		<div  class="logo"></div> 
+      <nav class="menu" id="menu">
+            
+        <div class="menu-close">
+            <a href="#" class="menu-close-icon"><img src="static/ic_menu.svg" alt="" srcset=""></a>
+        </div>
+        </nav>
+
+    </header>
+
+	<main>
+        <section class="bio">
+            
+            <h1 class="bio-header">
+                Bleu Yves <br> Sopoude<br>
+            
+            </h1>
+
+            <p class="bio-info">  
+            Hello! I am a software developer! I can help you build a product, feature or website. Take a look at my work. <br>  <br>
+
+            If you like what you see and have a project you need coded, don’t hesitate to contact me.
+            </p>
```

Comment:
- [ ] [Optional] I would suggest you please adjust the h1 tag as it is not showing on the page. and also this is your last review so I am pointing it as optional as there is padding on the left and right please adjust it to match the Figma design
### Your design
![his](https://user-images.githubusercontent.com/81727988/236179638-4b249466-f5a3-4de8-b638-1de90baf0fbc.png)

### Figma design
![figma](https://user-images.githubusercontent.com/81727988/236179665-be6287fe-0100-48de-b1c2-ac65564ec8da.png)